backend/tests/unit/dal/Services/ProjectServiceTest.cs

```diff
@@ -709,17 +755,27 @@ public void Update_AddBuilding()
             // Act
             var projectToUpdate = service.Get(project.ProjectNumber);
             projectToUpdate.AddProperty(building);
-            var result = service.Update(projectToUpdate);
+            service.Update(projectToUpdate);
+            var result = service.Get(project.ProjectNumber);
 
             // Assert
             Assert.NotNull(result);
-            Assert.Equal(projectToUpdate, result);
             Assert.Single(result.Properties);
+            result.Should().BeEquivalentTo(projectToUpdate, options => options
+            .IgnoringCyclicReferences()
+            .Excluding(x => x.SelectedMemberPath == "Properties[0].Building.Projects")
```

Comment:
While I like the ability to ignore problematic paths in general, I don't like how messy this makes the test. You can hide these excludes behind a configurationOptions object, but that could result in a lot of overhead for these sorts of highly nested objects. 

In the below example, I like this